src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin block($side) {
```

Comment:
The mixin name 'block' is too generic. Consider using a more specific name, such as 'calendar-cell'.